src/mainwindow.cpp

```diff
@@ -313,8 +313,8 @@ void MainWindow::setupKeyboardShortcuts()
     new QShortcut(QKeySequence(Qt::Key_Up), this, SLOT(selectNoteUp()));
     new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Enter), this, SLOT(setFocusOnText()));
     new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Return), this, SLOT(setFocusOnText()));
-    new QShortcut(QKeySequence(Qt::Key_Enter), this, SLOT(setFocusOnText()));
-    new QShortcut(QKeySequence(Qt::Key_Return), this, SLOT(setFocusOnText()));
+    //new QShortcut(QKeySequence(Qt::Key_Enter), this, SLOT(setFocusOnText()));
+    //new QShortcut(QKeySequence(Qt::Key_Return), this, SLOT(setFocusOnText()));
```

Comment:
The shortcut shouldn't have been commented for this PR

Reply:
Oh, true! Missed that one. @theshadowx Can you push a commit for that?

Reply:
Without it, 'Enter' key press won't work in the line-edit. Everytime you press enter, instead of searching it will go inside the text edit.